meilisearch-http/src/routes/stop_words.rs

```diff
@@ -1,63 +1,83 @@
-use std::collections::BTreeSet;
-
+use actix_web::{web, HttpResponse};
+use actix_web_macros::{delete, get, post};
 use meilisearch_core::settings::{SettingsUpdate, UpdateState};
-use tide::{Request, Response};
+use std::collections::BTreeSet;
 
-use crate::error::{ResponseError, SResult};
-use crate::helpers::tide::RequestExt;
-use crate::helpers::tide::ACL::*;
-use crate::routes::document::IndexUpdateResponse;
+use crate::error::ResponseError;
+use crate::helpers::Authentication;
+use crate::routes::{IndexParam, IndexUpdateResponse};
 use crate::Data;
 
-pub async fn get(ctx: Request<Data>) -> SResult<Response> {
-    ctx.is_allowed(Private)?;
-    let index = ctx.index()?;
-    let db = &ctx.state().db;
-    let reader = db.main_read_txn()?;
+pub fn services(cfg: &mut web::ServiceConfig) {
+    cfg.service(get).service(update).service(delete);
+}
+
+#[get(
+    "/indexes/{index_uid}/settings/stop-words",
+    wrap = "Authentication::Private"
+)]
+async fn get(
+    data: web::Data<Data>,
+    path: web::Path<IndexParam>,
+) -> Result<HttpResponse, ResponseError> {
+    let index = data
+        .db
+        .open_index(&path.index_uid)
+        .ok_or(ResponseError::index_not_found(&path.index_uid))?;
+    let reader = data.db.main_read_txn()?;
     let stop_words_fst = index.main.stop_words_fst(&reader)?;
     let stop_words = stop_words_fst.unwrap_or_default().stream().into_strs()?;
 
-    Ok(tide::Response::new(200).body_json(&stop_words).unwrap())
+    Ok(HttpResponse::Ok().json(stop_words))
```

Comment:
Why this instead of the `web::Json()` function?

Reply:
Like that, I always see at the end of the function the status code returned. And it's the same return as all handlers.